core/src/main/scala/org/apache/spark/rdd/NewHadoopRDD.scala

```diff
@@ -174,11 +174,8 @@ class NewHadoopRDD[K, V](
             }
           }
         } catch {
-          case e: Exception => {
-            if (!Utils.inShutdown()) {
-              logWarning("Exception in RecordReader.close()", e)
-            }
-          }
+          case e: Exception => 
+            logWarning("Exception in RecordReader.close()", e)
```

Comment:
I think we can back that out if there's any question to keep this limited. @JoshRosen Yes that's where we ended up again, now that it's clear that there are several ways and several places this can happen. It's easiest just to ignore the exception.